composer.json

```diff
@@ -19,7 +19,7 @@
     ],
     "require": {
         "php" : "^7.1",
-        "illuminate/support": "~5.5.0|~5.6.0",
+        "illuminate/support": "~5.5",
```

Comment:
If you could also update PHP version to `"php" : "^7.1.3"`, which is minimum requirement for both Laravel 5.6 and 5.7, but I must have forgot that when I updated the package to support Laravel 5.6

Reply:
While doing so, the line "Easy to use Laravel 5.5+" in the README should be updated to "Easy to use Laravel 5.6+" because PHP requirement advancement is incompatible with L5.5

Reply:
Great feedback, thank you. I'll tackle these ASAP.

Reply:
Latest commit addresses the requested changes. With the php version advancement to 7.1.3, I've removed 5.5 from illuminate/support. Please let me know if anything else needs attention.